dms/models/dms_file.py

```diff
@@ -377,10 +378,15 @@ def _compute_extension(self):
         for record in self:
             record.extension = file.guess_extension(record.name)
 
-    @api.depends("content")
+    @api.depends("content", "name")
     def _compute_mimetype(self):
+        """ Similar to addons/base/models/ir_attachment.py#L232 """
```

Comment:
This comment says nothing. And the line number can change. Please add a proper docstring and, if you need to refer to another resource, add a permalink.